.github/actions/install-briefcase/action.yml

```diff
@@ -141,6 +141,8 @@ runs:
 
         echo "Installing ${REPO}@${REF}"
         python -m pip install -U "git+${REPO}@${REF}"
+        # this command can fail for versions of Briefcase <= 0.3.16
+        python -m pip install -U "git+${REPO}@${REF}#subdirectory=automation" || true
 
         echo "Installed version: $(briefcase --version)"
         echo "version=${REF}" >> ${GITHUB_OUTPUT}
```

Comment:
hmm....this will be an interesting problem. The `install-briefcase` action usually runs to install from `main` or from a PR branch....but it can also be used to install previous releases of Briefcase. This will happen for PRs targeting the "version branches" on the templates repos....although, these PRs are not especially common.

Nonetheless, the Automation plugin will obviously not be available for these previous versions of Briefcase. So, while this command can fail and move on, we'll need to ensure that `app-build-verify` can still successfully run. This will effectively include ensuring that `enable-automation` is `false` and that `briefcase run` does not actually run.

ideas forthcoming...

Reply:
This strategy works...
- Ensure `python -m pip install` succeeds whether `briefcase-automation` can be installed or not
- Before building and running the apps, confirm whether `briefcase-automation` is installed
- Only run `briefcase run` if `briefcase-automation` could be installed

Reply:
See https://github.com/beeware/.github/pull/69#discussion_r1409618409 for continued discussion of backport PRs for templates.